dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -9,6 +9,7 @@ const enhanceCards = (collections: FEFrontCard[]): DCRFrontCard[] => {
 				format,
 				url: faciaCard.header.url,
 				headline: faciaCard.header.headline,
+				standfirst: faciaCard.card.trailText,
```

Comment:
This is a good indication that we probably should not rename.

Reply:
Yeah, and it's a nicer name too

Reply:
Yeah, and it's a nicer name too